robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -170,7 +172,8 @@ public InstrumentationConfiguration build() {
           "org.specs2",  // allows for android projects with mixed scala\java tests to be
           "scala.",      //  run with Maven Surefire (see the RoboSpecs project on github)
           "kotlin.",
-          "com.almworks.sqlite4java" // Fix #958: SQLite native library must be loaded once.
+          "com.almworks.sqlite4java", // Fix #958: SQLite native library must be loaded once.
+          "org.powermock"
```

Comment:
I think that packagesToNotAquire should just be simplified as all non-android classes, but again I think there is a leaky / undefined boundary by what is loaded by the instrumenting class loader. I think this needs fixing.